src/ScikitLearnAPI.jl

```diff
@@ -0,0 +1,52 @@
+
```

Comment:
Perhaps we could add some kind of acknowledgement here, assuming you agree it's accurate:
```suggestion
# The mapping of Julia functions into python functions in this file is adapted from code in ScikitLearn.jl, written by Cédric St-Jean. 
```